### PR TITLE
ci: move to github actions

### PR DESCRIPTION
**Pull request by nritsche**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      with:
+        python-version: 3.8
+
+    - name: Install pytest
+      run: |
+        pip install pytest
+
+    - name: Install chimedb
+      run: |
+        pip install .
```
Comment (anjakefala) [inline .github/workflows/main.yml:45]: Do you need to install with the `[tests]` extension? I think that causes problems with the new pip. =(

File: test/test_mediawiki.py [hunk 2]
```diff
@@ -6,33 +6,20 @@
...
-            db:              test
-            user_ro:         travis
-            passwd_ro:       ""
-            user_rw:         travis
-            passwd_rw:       ""
-            host:            127.0.0.1
-            port:            3306
-        """
-        )
+    os.close(fd)
```
Comment (anjakefala) [inline test/test_mediawiki.py:17]: Why did this change?

Comment (nritsche) [reply test/test_mediawiki.py:17]: So I don't have to create that MySQL db on github ci

File: .github/workflows/main.yml (hunk 1 above)
Comment (nritsche) [reply .github/workflows/main.yml]: It does seem to work

**Comment by ketiltrout** [comment]
@anjakefala do you think we can merge this?

**Review by anjakefala — APPROVED** [review]

**Pull request merged**